package.json

```diff
@@ -51,7 +51,7 @@
     "generate:deprecated-mods": "dotenv node built/src/generate-deprecated-mods.js",
     "generate:weapons-from-quest": "dotenv node built/src/generate-weapons-from-quest.js",
     "generate:voice-dim-perk-names": "dotenv node built/src/generate-perk-names-for-voiceDIM.js",
-    "generate-data": "run-s build generate:*"
+    "generate-data": "run-s build generate:* generate:*"
```

Comment:
something is being generated out of order, that running this a second time fixes... should track that down...